lib/sidekiq_unique_jobs/locksmith.rb

```diff
@@ -331,6 +333,20 @@ def write_lock_info(conn)
       conn.set(key.info, lock_info)
     end
 
+    #
+    # Writes lock score to redis.
+    #   The lock score is used to improve performance when iterating large sorted
+    #   sets e.g the schedule and retry sets.
+    #
+    #
+    # @return [void]
+    #
+    def write_lock_score(conn)
+      return unless lock_score?
+
+      conn.set(key.score, lock_score)
```

Comment:
Need to add tests for when this gets set, i.e. only for the `schedule` and `retry` queues.